docs/how-to-guides/others/applying-clang-tidy-to-ros-packages.md

```diff
@@ -12,17 +34,29 @@ colcon build --cmake-args -DCMAKE_EXPORT_COMPILE_COMMANDS=1
 
 ## Usage
 
+If you are on Ubuntu 22.04, run the following command **only once** to help `run-clang-tidy` find the correct include paths.
+**Reference:** [autoware/pull/5543](https://github.com/autowarefoundation/autoware/pull/5543#issuecomment-2533325979).
+
+```bash
+sed -i '/- -Wno-c11-extensions/a\  - -I/usr/include/c++/11\n  - -I/usr/include/x86_64-linux-gnu/c++/11' .clang-tidy-ci
+```
+
+To run Clang-Tidy with all files in a package and export fixes to a YAML file, run the following command from the workspace root.
+Here we use `autoware_utils` as an example package name.
+
 ```bash
-clang-tidy -p build/ path/to/file1 path/to/file2 ...
+cd autoware/
+run-clang-tidy -p build/ -config="$(cat .clang-tidy-ci)" -export-fixes clang-tidy-fixes.yaml -j $(nproc) $(fdfind -e cpp -e hpp --full-path "/autoware_utils/") > clang-tidy-report.log
```

Comment:
Why are you using `.clang-tidy-ci` instead of `.clang-tidy`?

Reply:
Because this is the requirement for approval of CI, so when people are confused why it fails and want to pass CI, they can run this.

Reply:
https://github.com/autowarefoundation/autoware_universe/blob/08af8210f9cd51c0eeb52b588392ffd59b7ad02b/.github/workflows/clang-tidy-differential.yaml#L89

This is what we check for approval.